ocs_ci/ocs/resources/pod.py

```diff
@@ -455,3 +456,20 @@ def get_osd_pods(osd_label=constants.OSD_APP_LABEL, namespace=None):
     osds = get_pods_having_label(osd_label, namespace)
     osd_pods = [Pod(**osd) for osd in osds]
     return osd_pods
+
+
+def get_pod_obj(name, kind, namespace=None):
+    """
+    Returns the pod obj for the given pod
+
+    Args:
+        name (str): name of the resources
+        kind (str): kind of the resources
+
+    Returns:
+        obj : A pod object
+    """
+    ocp_obj = OCP(api_version='v1', kind=kind, namespace=namespace)
```

Comment:
kind is always Pod?

Reply:
yeah, I'll change accordingly

Reply:
please fix this function and ping me to review again:
it should not get kind as it is always pod

Reply:
Apologies, I missed to address in my previous commit and changed now 'kind=constants.POD'